.github/workflows/gradle.yml

```
@@ -93,6 +93,15 @@ jobs:
           SPRING_PROFILES_ACTIVE: ${{ matrix.schema}}
         run: ./gradlew :${{matrix.project}}:build --scan ${{ secrets.GRADLE_ARGS }}
 
+      - name: Execute Gradle using transaction executor
+        if: ${{ matrix.project == 'web3'}}
```

As mentioned previously, we can just test v1 for now to save execution time.

```suggestion
        if: ${{ matrix.project == 'web3' && matrix.schema == 'v1'}}
```

---

added